src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx

```diff
@@ -52,6 +52,9 @@ class CC32xxSocket;
 class CC32xxWiFi : public Singleton<CC32xxWiFi>
 {
 public:
+    /** the value passed to wlan_profile_del() to remove all profiles */
+    static constexpr int PROFILE_DELETE_ALL = 0xFF;
```

Comment:
I don't see an implementation for this feature. Does this come from the CC32xx network API or would we have to write some code for this to work?